tox.ini

```diff
@@ -19,4 +20,5 @@ deps =
 commands =
        pytest -n {posargs} -s -v --timeout=300 --reruns=2 --reruns-delay=15 \
               --cov=awswrangler --cov-report=xml --cov-report term-missing --cov-branch \
+              --cov-fail-under={[testenv]cov_fail_under} \
```

Comment:
Distributed CB is likely to fail since its coverage is lower

Reply:
I thought this way of defining `cov_fail_under` would allow me to override the value for the distributed environment. I realized that I had to use `setenv` instead.

With the latest changes, I will simply add the following line to the [tox.ini](https://github.com/aws/aws-sdk-pandas/blob/release-3.0.0/tox.ini#L25) file in the `release-3.0.0` branch:
```
setenv =
       COV_FAIL_UNDER = XX  # whatever number this ends up being
```